.github/update-sync-param-files.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import glob
+
+import yaml
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("output", type=str, help="output path of sync-param-files.yaml")
```

Comment:
If you use the word `output`, I think it's better also to prepare the arg `input`.
If not, I feel `target` or something like that is a better naming.